jsapp/js/components/common/miniAudioPlayer.scss

```diff
@@ -10,6 +10,10 @@
   flex-wrap: nowrap;
   color: colors.$kobo-blue;
   height: button.$button-height-s;
+
+  > audio {
```

Comment:
Since I can't see the double click error what this is targeting? Nothing breaks or looks different, just curious

Reply:
We've been rendering our own play button for the audio player, and we still are. This rule prevents a browser from displaying a built-in widget, now that we're actually attaching an `<audio>` tag to the DOM.